### ISSUE TITLE
Populate learning objectives across all episodes

### ISSUE THREAD

**Issue by ragamouf** [LibraryCarpentry/lc-fair-research#9]
Determining learning objectives will help how the content on each page is shaped. 

Learning objectives could be drawn from any number of existing FAIR training materials.

**Comment by ragamouf** [comment]
I've submitted a pull request with learning objectives for eps 2-5 (ie F,A,I,R) and assigned @kmhettne as the reviewer - does that work as a convention?

**Comment by linajandren** [comment]
I've been looking through the learning objectives posted this morning and I think they are a great start but we need to think a little more about how to phrase them. Most of the learning objectives, especially for Findable and Accessible are phrased as something the learner should be able to do after completing the course. (Example: Articulate the role of data repositories in enabling FAIR data - I could say "After the course, the learner should be able to articulate the role of data....") Some though, especially for the Interoperable episode, are phrased as things to do during the course (Example: Demystify the term 'interoperability' - if I want the learner to be able to do that after the workshop that would be rather high level). I guess it comes down to if the objectives are written as learner objectives or course objectives, and both could be done. But right now it's a little bit of both, and I believe it would be better if we kept to one of them.

**Comment by kmhettne** [comment]
I have opened issues to discuss the learning objectives per Episode. That might ease the way of working with them.

**Comment by kmhettne** [comment]
As a reminder: Learning objectives are the highest level and focus on what the learner will learn (see comment from Liz in issue #8 ).

**Comment by ragamouf** [comment]
I agree that it's useful to disambiguate the overall course or lesson objectives from the individual objectives for each episode.  I don't really mind if we call them learning objectives or leearner objectives. The purpose at the episode level is to indicate the work and activities ahead in that particular episode. 

So i think we should take it on to ensure that the objectives within each episode stay within those bounds, and we can amend these where it's appropriate to do so.
